source/includes/_tipser-elements.md

```diff
@@ -112,6 +112,13 @@ Insert the following HTML snippet in your code in the place where you want the `
 <div id="tipser_store"></div>
 ```
 
+For more fitting mobile experience you can choose between two types of mobile menu. The default one is a dropdown with categories. 
+The optional menu renders inline menu items, and is applied by adding `data-tipser-inlineMenu` attribute to the HTML snippet:
+
+```html
+<div id="tipser_store" data-tipser-inlineMenu></div>
```

Comment:
chyba lepsza byłaby nazwa `data-tipser-inline-menu`, ale skoro już jest snake case to niech zostanie